packages/tokamak/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -157,24 +149,26 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
         return systemConfig;
     }
 
-    /// @notice Getter for the Guardian
+    /// @notice Getter function for the address of the guardian.
+    ///         Public getter is legacy and will be removed in the future. Use `SuperchainConfig.guardian()` instead.
+    /// @return Address of the guardian.
     /// @custom:legacy
     function GUARDIAN() external view returns (address) {
-        return guardian;
+        return guardian();
     }
 
-    /// @notice Pauses withdrawals.
-    function pause() external {
-        require(msg.sender == guardian, "OptimismPortal: only guardian can pause");
-        paused = true;
-        emit Paused(msg.sender);
+    /// @notice Getter function for the address of the guardian.
+    ///         Public getter is legacy and will be removed in the future. Use `SuperchainConfig.guardian()` instead.
+    /// @return Address of the guardian.
+    /// @custom:legacy
+    function guardian() public view returns (address) {
+        return superchainConfig.guardian();
```

Comment:
This function returns the zero address when running devnet script

Reply:
Thanks for checking. Let me check it as well.